pkg/gadgets/trace/dns/tracer/bpf/dns.c

```diff
@@ -105,6 +105,7 @@ int ig_trace_dns(struct __sk_buff *skb)
 		return 0;
 
 	struct event_t event = {0,};
+	event.timestamp = bpf_ktime_get_ns();
```

Comment:
Is this the right helper to use? Wasn't sure if `bpf_ktime_get_coarse_ns` or `bpf_ktime_get_boot_ns` might be more appropriate.